src/app/pages/event/preview/index.vue

```diff
@@ -0,0 +1,396 @@
+                      <div
+                        v-if="form.previewUrls.length > 3"
+                        class="group absolute inset-0 flex cursor-pointer items-center justify-center bg-black/50"
+                      >
+                        <div class="text-center text-white">
+                          <div class="mb-1 flex items-center justify-center">
+                            <Grid class="h-6 w-6" />
+                          </div>
+                          <span class="text-sm font-medium">{{
+                            t('seeAll')
+                          }}</span>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div
+              v-else
+              class="rounded-2xl border-2 border-dashed border-gray-300 p-12 text-center"
+            >
+              <div class="flex flex-col items-center">
+                <Image class="mb-4 h-12 w-12 text-gray-400" />
+                <p class="text-gray-500">{{ t('noHighlights') }}</p>
+              </div>
+            </div>
+          </div>
+          <div>
+            <h2 class="mb-3 text-2xl font-bold">{{ t('organizer') }}</h2>
+            <div class="flex items-center gap-4">
+              <AccountProfilePicture
+                :account-id="store.signedInAccountId || ''"
+                classes="h-16 w-16 rounded-full bg-gray-100"
+                height="64"
+                width="64"
+              />
+              <h3 class="text-xl font-semibold text-[#5B47D9]">
+                {{ store.signedInUsername }}
+              </h3>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="flex flex-col gap-4 bg-(--semantic-faint-line) py-4">
+      <ButtonColored
+        variant="primary"
+        class="w-full rounded-lg"
+        :aria-label="t('publish')"
+        @click="createEvent"
+      >
+        {{ t('publish') }}
+      </ButtonColored>
+      <ButtonColored
+        variant="secondary"
+        class="w-full rounded-xl"
+        :aria-label="t('saveDraft')"
+        @click="handleDraftSave"
+      >
+        {{ t('saveDraft') }}
+      </ButtonColored>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref } from 'vue'
+import { Share, Heart, MoreHorizontal, Grid, Image } from 'lucide-vue-next'
+import {
+  Carousel,
+  CarouselContent,
+  CarouselItem,
+} from '@/components/scn/carousel'
+import type { EventStorageStrategy } from '~/types/events/EventStorageStrategy'
+import { LocalStorageStrategy } from '~/utils/storage/LocalStorageStrategy'
+import { useCreateEventMutation } from '~~/gql/documents/mutations/event/eventCreate'
+import { useUploadCreateMutation } from '~~/gql/documents/mutations/upload/uploadCreate'
+import { EventVisibility } from '~~/gql/generated/graphql'
+import Tus from '@uppy/tus'
+import Uppy from '@uppy/core'
```

Comment:
```suggestion
import Tus from '@uppy/tus'
import Uppy from '@uppy/core'
import { Share, Heart, MoreHorizontal, Grid, Image } from 'lucide-vue-next'

import type { EventStorageStrategy } from '~/types/events/EventStorageStrategy'

import { useCreateEventMutation } from '~~/gql/documents/mutations/event/eventCreate'
import { useUploadCreateMutation } from '~~/gql/documents/mutations/upload/uploadCreate'
import { EventVisibility } from '~~/gql/generated/graphql'
```

The `storage` directory can be exported in `src/app/utils/index.ts` for the auto-import to become available for `LocalStorageStrategy`.